src/components/ActionSheetItem/ActionSheetItem.tsx

```diff
@@ -84,77 +82,44 @@ const ActionSheetItemComponent = ({
           ? onClick
           : onItemClick(onClick, onImmediateClick, Boolean(autoclose))
       }
-      activeMode="ActionSheetItem--active"
-      // eslint-disable-next-line vkui/no-object-expression-in-arguments
+      activeMode={
+        platform === Platform.IOS ? "ActionSheetItem--active" : undefined
+      }
       vkuiClass={classNames(
         getClassName("ActionSheetItem", platform),
         `ActionSheetItem--${mode}`,
         `ActionSheetItem--sizeY-${sizeY}`,
-        {
-          "ActionSheetItem--compact": isCompact,
-          "ActionSheetItem--desktop": isDesktop,
-          "ActionSheetItem--withSubtitle": hasReactNode(subtitle),
-        }
+        isCompact && "ActionSheetItem--compact",
+        isDesktop && "ActionSheetItem--desktop",
+        hasReactNode(subtitle) && "ActionSheetItem--withSubtitle"
       )}
       Component={Component}
     >
       {hasReactNode(before) && (
         <div vkuiClass="ActionSheetItem__before">{before}</div>
       )}
-      <div vkuiClass="ActionSheetItem__container">
+      <div
+        vkuiClass={classNames(
+          "ActionSheetItem__container",
+          !multiline && "ActionSheetItem--text-ellipsis"
```

Comment:
Вижу, что у нас сделано похожим образом в `RichCell`, но в основном все-таки используется такой подход:
<img width="243" alt="image" src="https://user-images.githubusercontent.com/22026957/179951013-93ab81b2-4746-413c-9b73-761c8339ff77.png">

У меня два варианта того, как поступить:
1. переписать тут "как везде" и отрицание вносить уже в стилях через `:not(.ActionSheetItem--multiline)` 😉
```suggestion
          multiline && "ActionSheetItem--multiline"
```

2. оставить как есть, разве что я бы предложила класс подсократить, и накинуть задачку порефачить везде, где есть, `multiline`, на `!multiline && "Component--ellipsis"`. Профит вижу в том, что мы чутка сократим стили.
```suggestion
          !multiline && "ActionSheetItem--ellipsis"
```

Reply:
Наверное, нужно ещё подождать мнения других, но, мне кажется, что проще накинуть парочку свойств через новый класс, чем париться со специфичностью и "занулять" свойства, т.е. я за вариант 2

Reply:
Да, лучше по второму пути

Правда в сокращении имен модификаторов, т.к. теряется семантика, `ellipsis` слишком обобщённо. Мне нравится подход в БЭМ с `<key>-<value>`.

Reply:
> Я за первый вариант. `multiline` выглядит как особый кейс элемента поэтому логичнее будет сделать стили по умолчанию для `ellipsis` и добавлять `multiline`

Тут с какой стороны посмотреть, с точки зрения дизайна это особый кейс, но в вебе по умолчанию `multiline`, а `ellipsis` нужно дополнительно прикручивать 🙂